### ISSUE TITLE
MZmine 2.41.2 crash 3D visualizer

### ISSUE THREAD

**Issue by pvervliet** [mzmine/mzmine2#672]
### Setup

I am reporting a problem with MZmine 2 version 2.41.2. 
My operating system is Windows 10 Enterprise.

Below I pasted the exact output of the following commands:
```
java -version
java version "1.8.0_221"
Java(TM) SE Runtime Environment (build 1.8.0_221-b11)
Java Hotspot(TM) 64-bit Server VM (build 25.221-b11, mixed mode)

R --version
R version 3.6.0 (2019-04-26) -- "Planting of a Tree"
Copyright (C) 2019 The R Foundation for Statistical Computing
Platform: x86_64-w64-mingw32/x64 (64-bit)
```

### Expected behavior
Import raw data (mzdata.xml)
Create 3D visualization of MS1 for auto m/z range.

### Actual behavior
MZmine closes abruptly after confirming parameters for the 3D visualization. 
In the command window following messages are present:

javax.media.j3d.IllegalRenderingStateException: Java 3D ERROR : OpenGL 1.2 or better is required (GL_VERSION=1.1)
        at javax.media.j3d.JoglPipeline.setupCanvasProperties(JoglPipeline.java:8090)
        at javax.media.j3d.JoglPipeline.access$600(JoglPipeline.java:88)
        at javax.media.j3d.JoglPipeline$ContextQuerier.init(JoglPipeline.java:8719)
        at javax.media.j3d.JoglPipeline$QueryCanvas.doQuery(JoglPipeline.java:8626)
        at javax.media.j3d.JoglPipeline$QueryCanvas.access$100(JoglPipeline.java:8574)
        at javax.media.j3d.JoglPipeline.createQueryContext(JoglPipeline.java:6561)
        at javax.media.j3d.Canvas3D.createQueryContext(Canvas3D.java:4619)
        at javax.media.j3d.Canvas3D.createQueryContext(Canvas3D.java:3616)
        at javax.media.j3d.Renderer.doWork(Renderer.java:461)
        at javax.media.j3d.J3dThread.run(J3dThread.java:271)

DefaultRenderingErrorListener.errorOccurred:
CONTEXT_CREATION_ERROR: Renderer: Error creating Canvas3D graphics context for queryProperties()
This version of Java3D can't query "textureWidthMax/textureHeightMax"
so they are being assigned the default values:
textureWidthMax:  1024
textureHeightMax:  1024graphicsDevice = D3DGraphicsDevice[screen=0]

If images render as a 'grey-box', try setting these parameters
to a lower value, eg. 512, with '-DtextureWidthMax=512'
Otherwise check your graphics environment specificationscanvas = javax.media.j3d.Canvas3D[canvas0,0,0,0x0,invalid]

[15:07:42|INFO|net.sf.mzmine.main.impl.MZmineConfigurationImpl]: Saved configuration to file conf\config.xml

### Steps to reproduce the problem
After import of data, right click and select 'Show 3D visualizer'
Select raw data file
Scans: Leave default parameter (only MS1)
m/z: Auto range (=> 55.0501 - 998.7172) in specific data file
Retention time resolution: 500
m/z resolution: 500

**Comment by tomas-pluskal** [comment]
Thanks for the report. The current 3D visualizer is based on outdated Java3D library that is not supported anymore. We are working on a new 3D visualizer module that is based on JavaFX and will fix these issues (https://github.com/Akshaj5/mzmine2/tree/new3d/src/main/java/net/sf/mzmine/modules/visualization/fx3d). It should be finished in a few weeks.

**Comment by tomas-pluskal** [comment]
Fixed in MZmine version 2.50
